package.json

```diff
@@ -293,7 +293,9 @@
     "xml2js": "^0.6.2",
     "tough-cookie": "4.1.3",
     "semver": "^7.6.3",
-    "superagent": "^10.2.2"
+    "superagent": "^10.2.2",
+    "min-document@npm:*": "npm:2.19.1",
+    "global@npm:*": "npm:4.4.1"
```

Comment:
Confused by this - just checked global npm web page and the current version is 4.4.0 https://www.npmjs.com/package/global?activeTab=versions